dragonfly/implementations/daq_run_interface.py

```diff
@@ -332,4 +343,613 @@ def save_trace(self, trace, comment):
             logger.debug("things are about to be dumped in file")
             json.dump(result_meta, outfile, indent=4)
             logger.debug("things have been dumped in file")
-        logger.info("saving {}: successful".format(path))
+            logger.info("saving {}: successful".format(path))
+
+
+
+__all__.append('ROACH1ChAcquisitionInterface')
```

Comment:
Let's split this class into its own file, it is probably fine for the single channel and multi-channel roach classes to be in the same file. Also, if there area methods in common between the two then perhaps one could inherit from the other, or there should be a base class that they both extend (I haven't looked through the details enough to be sure if that makes sense).